### ISSUE TITLE
Shipping symbols with RenderDoc

### ISSUE THREAD

**Issue by CookiePLMonster** [baldurk/renderdoc#929]
Are there any plans to start shipping PDB files for injector DLLs, like renderdoc.dll? Installing v1.0 does not provide PDBs, while I have noticed my RenderDoc dir **does** contain them - however, they date back to Feb 2018 and not Mar 2018, so they are a leftover from an older version and not provided anymore.

Providing symbols with released binaries without asking the users to build everything by themselves (I am 100% certain I have never built RenderDoc on my own, so those PDB files must have been shipped with some older version) may prove useful for debugging. For example, right now I am dealing with an OpenGL game which crashes at a specific spot for a noticeable amount of people, with crashes always being somewhere inside a graphics driver DLL. However, while capturing frames from the game via RenderDoc works fine by the time the game starts, game **crashes** when trying to capture a frame from the spot users crash at. Crash itself traces to renderdoc.dll, with in-game GL calls a few calls down the stack - so I expect knowing why exactly RenderDoc crashes here will help identify the issue and further decide whether the crash is a bug in RenderDoc (not so likely) or the game itself (more likely).

Cheers!

**Comment by baldurk** [comment]
Yeh in previous .zip releases of RenderDoc (which were used for auto-updates so even people who originally installed with the .msi would have gotten them) I had the symbols, but I realised that the symbols were inflating the download and storage sizes for everyone while only being useful for a minority of users.

So instead I split off the symbols and set up a regular [symbol server](https://renderdoc.org/symbols) like Microsoft, Mozilla and others have. I did set up a [wiki page](https://github.com/baldurk/renderdoc/wiki/Symbol-Server) about it but I definitely should do more to try and highlight its existence. At least a link on the README and maybe a link on the downloads page too.
